aztec/src/main/kotlin/org/wordpress/aztec/toolbar/AztecToolbar.kt

```diff
@@ -312,7 +312,7 @@ class AztecToolbar : FrameLayout, OnMenuItemClickListener {
         return editor != null && editor is AztecText
     }
 
-    fun setEditor(editor: AztecText, sourceEditor: SourceViewEditText) {
+    fun setEditor(editor: AztecText, sourceEditor: SourceViewEditText?) {
```

Comment:
All of those occurrences where in `toggleEditorMode`, so I decided to just check for `sourceEditor` presence right away on that method, here 31fa909